### ISSUE TITLE
EVA saying "Welcome Back Commaner" on first run

### ISSUE THREAD

**Issue by fruestueck** [OpenRA/OpenRA#14881]
After a fresh TS installation [this video](https://www.youtube.com/watch?v=N75H5amC8Is) playse once at the first startup, saying `welcome back commander` at the end.

The could be a nice additon to OpenRA, at the first startup after a installation.

**Comment by GSonderling** [comment]
Would it be ok from copyright pow?

**Comment by MunWolf** [comment]
The data for TS was released as freeware from EA, not sure if it is abandonware yet but it should be fine.

**Comment by fruestueck** [comment]
Thanks for the correction reaperrr, didn't see it 👍 

If copyright allows, we could add it into the gamemod, otherwise it could be imported from SUN/Firestorm Addon disc like the rest

**Comment by dan9550** [comment]
I like this idea. IIRC TD and RA had something like this, but it may have been the games jumping to the first "GoodGuys" mission.

**Comment by fruestueck** [comment]
We could use the same "Welcome Back Commander" Eva Message for TD and TS. For RA there surely is a similar message somewhere hidden in the audio.mix

**Comment by dan9550** [comment]
@fruestueck Would not be for using the TS assets in TD personally as its not canon. If you wanted to get nitty gritty into the story and lore I'm sure that incarnation of EVA technically didn't exist in the TD days.
That's just my two cents 😊

**Comment by dan9550** [comment]
Alright is nearly 1 am I am trying to sleep and I couldn't get this off my mind. I was right in remembering there was an intro video for TD so if this is what you were referring too I'm all for it.
https://youtu.be/MEfvfP4aMF0

Red Alert I believe played the briefing video for the first allied mission after the setup. The setup can be seen here https://youtu.be/Ri-VlJVxc-g

**Comment by fruestueck** [comment]
> TD https://youtu.be/MEfvfP4aMF0
This video was made by capturing the installation process of TD.
> Red Alert https://youtu.be/Ri-VlJVxc-g
Same here.

Initially i meant to show a breav "welcome back commander" on each mod after first startup but your idea of keeping more vanilla and showing the original behavior (e.g. RA playing the first briefing video) is more favorable. These caputred installation videos would then not be shown exept we decide to play them while installing OpenRA or we decide to play them also at the first launch of a MOD.

To sum it up, I'd really like a short "Welcome Back Commander" message that doesn't get skipped if the user skips an intro video before.

I could upload the soundfiles and vids i've found on request

**Comment by fruestueck** [comment]
The Textanimation from #17057 could be used to write a "welcome back commander" message with black background before main menu.

And with it playback of `00-w000.wav` (eva, [▻Play](https://www.dropbox.com/s/6w984linv1is1ug/00-w000.wav?dl=0)) or `01-w000.wav` (cabal) from `wdtvox.mix` (XCC Mixer)

Would that be suitable

**Comment by AlexanderRavenheart** [comment]
This reveal animation can now be achieved with the latest update to RevealingLabelWidget. You can specify reveal mode: Blank, Underline, Block. The reveal speed is determined by RevealStep and RevealCyclesDelay attributes.
